src/rt-timeseries/RtTimeseriesContainer.tsx

```diff
@@ -9,7 +10,14 @@ interface Props {
 }
 
 const RtTimeseriesContainer: React.FC<Props> = ({ data }) => {
-  return data ? <RtTimeseries data={data} /> : <Loading />;
+  const notEnoughData = data && data.Rt.length < 2;
+  return data ? (
+    notEnoughData ? null : (
```

Comment:
it's a function of how many model versions have been stored for this facility, and also of various design decisions we made about the chart. There might be another solution for this use case eventually but for now I'm just trying to avoid showing people a broken chart.

There isn't any de-duping currently, as was discussed in https://github.com/Recidiviz/covid19-dashboard/commit/191e4b09ad699b7941fa012b94b1ab9e58362085#r38825212 ... it is unfortunate that changes to data that is not part of the model causes a new model version to be saved, but it does appear that that's the case; I hadn't realized that before.

Already created #254 to try to clean that up; I need to confirm how the Python function handles same-day multiples in order to figure out the best course of action for this use case.